draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3583,7 +3580,7 @@ Procedure:
 3.  r = r * x
 4.  b = t
 5.  c = c5
-6.  for k in (m, m - 1, ..., 2):
+6.  for k in (c1, c1 - 1, ..., 2):
```

Comment:
Input: x, an element of F.
Output: r, an element of F such that r^2 == x.

Reply:
I don't follow?

Reply:
Oh, you just want to rename. (This comment doesn't apply to the current line.)